src/pyuvdata/telescopes.py

```diff
@@ -511,7 +679,11 @@ def update_params_from_known_telescopes(
         run_check: bool = True,
         check_extra: bool = True,
         run_check_acceptability: bool = True,
-        known_telescope_dict: dict = _KNOWN_TELESCOPES,
+        feeds: str | list[str] | None = None,
+        polarization_array: np.ndarray | None = None,
+        flex_polarization_array: np.ndarray | None = None,
+        override_known_params: bool = True,
+        **kwargs,
```

Comment:
should this kwargs be documented? I don't know what can be passed to it...

Reply:
Added in 1b8340609b362fc9b9225a3eb7e9535b7793952d.